package.json

```diff
@@ -29,11 +32,17 @@
     "@ethersproject/keccak256": "^5.7.0"
   },
   "devDependencies": {
-    "@types/mocha": "^10.0.0",
+    "@fast-check/ava": "^1.2.1",
+    "@typescript-eslint/eslint-plugin": "^7.1.0",
+    "@typescript-eslint/parser": "^7.1.0",
+    "ava": "^6.1.1",
     "c8": "^7.12.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.1.3",
     "fast-check": "^3.3.0",
-    "mocha": "^10.1.0",
     "prettier": "^3.2.5",
+    "rimraf": "^5.0.5",
```

Comment:
Is `rimraf` needed? I know is widely used but reducing the surface may be a better idea.

Reply:
It replaces `rm -rf ...`. I think its better for portability.